gateway/tests/api/test_v1_program.py

```diff
@@ -94,6 +113,57 @@ def test_upload_private_function(self):
         self.assertEqual(programs_response.status_code, status.HTTP_200_OK)
         self.assertEqual(programs_response.data.get("provider"), None)
 
+    def test_upload_custom_image_without_provider(self):
+        """Tests upload end-point authorized."""
+
+        user = models.User.objects.get(username="test_user_2")
+        self.client.force_authenticate(user=user)
+
+        env_vars = json.dumps({"MY_ENV_VAR_KEY": "MY_ENV_VAR_VALUE"})
+        programs_response = self.client.post(
+            "/api/v1/programs/upload/",
+            data={
+                "title": "Private function",
+                "entrypoint": "test_user_2_program.py",
```

Comment:
`entrypoint` is not included in the data for the upload with the container image.

Reply:
Solved [e4558a3](https://github.com/Qiskit/qiskit-serverless/pull/1345/commits/e4558a3c702c82949d1855b0d98eb30c49e51477)